packages/react-instantsearch/src/connectors/connectHierarchicalMenu.test.js

```diff
@@ -130,7 +130,7 @@ describe('connectHierarchicalMenu', () => {
     expect(props).toBe(null);
   });
 
-  it('calling refine updates the widget\'s state', () => {
+  it('calling refine updates the widget\'s searchState', () => {
```

Comment:
"calling refine updates the search state", ultimately the widget has no state, there's only the search state